cmd/internal/applicationsnapshot/report_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright © 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package applicationsnapshot
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	rs "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"github.com/stretchr/testify/assert"
+)
+
+//go:embed test_snapshot.json
+var jsonstr string
```

Comment:
Perhaps a more indicative name?

```suggestion
var testSnapshot string
```